andela/ah-frontend-tesseract
# #160814327 comment on articles 160814327

#### What does this PR do?
implements crud functionality for commenting on articles
#### Description of Task to be completed?
- create a comment to an article
- edit a comment you have created
- delete a comment you have created
#### How should this be manually tested?
- run the application using `npm start`
- navigate to articles and click on any of the articles 
- if logged in you will be able to comment on an article and you will be able to see a list of articles
#### What are the relevant pivotal tracker stories?
#160814327
#### Screenshots
<img width="997" alt="screen shot 2018-10-26 at 09 56 36" src="https://user-images.githubusercontent.com/30045448/47549688-825d0c00-d905-11e8-9f8e-6ba9a93ae863.png">
